README.md

```diff
@@ -27,11 +27,11 @@ Where data owners and data scientists can securely collaborate without exposing
 
 This wasn’t possible until now for highly regulated fields like health, finance, or advertising. When collaborating remotely, data owners had to open their whole dataset, often through a Jupyter notebook. This was dangerous because too many operations were allowed and the data scientist had numerous ways to extract information from the remote infrastructure (print the whole database, save the dataset in the weights, etc).
 
-BastionLab solves this problem by ensuring that only privacy-friendly operations are allowed on the data and anonymized outputs are shown to the data scientist. 
+BastionLab solves this problem by ensuring that only privacy-friendly operations are allowed on the data and aggregated outputs are shown to the data scientist. 
 
 **BastionLab is an open-source project.** Our solution is coded in Rust and uses Polars, a pandas-like library for data exploration.
 
->  **Disclaimer**: BastionAI, our fortified learning framework using TEEs will be incorporated in the broader offers of BastionLab, a holistic secure data science toolkit. While waiting for the merge, you can still use BastionAI under the folder bastionai.
+>  **Disclaimer**: BastionAI, our fortified learning framework relying on Trusted Execution Environement (TEE) will be incorporated in the broader offers of BastionLab. While waiting for the merge, you can still use BastionAI under the folder bastionai.
```

Comment:
As the integration is complete we can probably change this and present the AI part

Reply:
Ah? Is that the official position from Daniel also? If not, I think for now, let's just keep it and see in the next PR? ^^